hms-lambda-handler/src/main/java/com/amazonaws/athena/hms/handler/CreateTableHandler.java

```diff
@@ -45,6 +46,9 @@ public CreateTableResponse handleRequest(CreateTableRequest request, Context con
       TDeserializer deserializer = new TDeserializer(getTProtocolFactory());
       Table table = new Table();
       deserializer.fromString(table, request.getTableDesc());
+      if (S3ASchemeUpdater.isTableUsingS3Scheme(table)) {
```

Comment:
Can we add a comment here explaining why we do this in-place conversion?

Reply:
Thanks CY. Will add the comment.